src/components/Controls.jsx

```diff
@@ -26,12 +27,14 @@ class Controls extends React.Component {
    */
   handlePrevClick () {
```

Comment:
You are duplicating this with handleNextClick. Think about extracting this pattern to a pure function.